build.gradle.kts

```diff
@@ -32,18 +34,25 @@ dependencies {
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.2")
     implementation("com.google.code.findbugs:jsr305:3.0.2")
     implementation("org.apache.httpcomponents.client5:httpclient5:5.2.1")
+
+    testImplementation("org.mock-server:mockserver-netty:5.15.0")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.7.1")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+
 }
 
 val gradleEnterpriseVersion = "2023.2" // Must be later than 2022.1
 val baseApiUrl = providers.gradleProperty("apiManualUrl").orElse("https://docs.gradle.com/enterprise/api-manual/ref/")
 
 val apiSpecificationFileGradleProperty = providers.gradleProperty("apiSpecificationFile")
+val apiSpecificationURL = "${baseApiUrl.get()}gradle-enterprise-${gradleEnterpriseVersion}-api.yaml"
```

Comment:
Calling get() defeats the purpose of the Provider API. Please rewrite the code so it remains Provider-API-based. If you get stuck, @wolfs can surely help.